### DISCUSSION TITLE
Create Views

### DISCUSSION THREAD

**Discussion by darkalchemy** [kannibalox/pyrosimple#71] [Q&A]
Is the an easy way to create custom views based on rutorrent labels or by tracker, etc?

Thanks

**Comment by kannibalox** [comment]
The `--to-view` flag (and all the other view-related flags) of `rtcontrol` will automatically create a view if it doesn't exist: 
```bash
rtcontrol label=movies --to-view movies
```

If you're looking for permanent views that get updated automatically, that's better done through the config with commands like `view.add` and `view.filter`.
